src/net_processing.cpp

```diff
@@ -88,19 +88,19 @@ static constexpr int32_t MAX_PEER_TX_REQUEST_IN_FLIGHT = 100;
  *  the actual transaction (from any peer) in response to requests for them. */
 static constexpr int32_t MAX_PEER_TX_ANNOUNCEMENTS = 5000;
 /** How long to delay requesting transactions via txids, if we have wtxid-relaying peers */
-static constexpr auto TXID_RELAY_DELAY = std::chrono::seconds{2};
+static constexpr auto TXID_RELAY_DELAY{2s};
 /** How long to delay requesting transactions from non-preferred peers */
-static constexpr auto NONPREF_PEER_TX_DELAY = std::chrono::seconds{2};
+static constexpr auto NONPREF_PEER_TX_DELAY{2s};
 /** How long to delay requesting transactions from overloaded peers (see MAX_PEER_TX_REQUEST_IN_FLIGHT). */
-static constexpr auto OVERLOADED_PEER_TX_DELAY = std::chrono::seconds{2};
+static constexpr auto OVERLOADED_PEER_TX_DELAY{2s};
 /** How long to wait (in microseconds) before downloading a transaction from an additional peer */
-static constexpr std::chrono::microseconds GETDATA_TX_INTERVAL{std::chrono::seconds{60}};
+static constexpr auto GETDATA_TX_INTERVAL{60us};
```

Comment:
This is wrong 60s is not equal to 60us

Reply:
Sorry. I made an error interpreting this line of code earlier.

In the updated PR (commit 16d0b619d72971b23eabcb0a5849370720f93909), I have changed the variable to **std::chrono::seconds.**

`std::chrono::microseconds GETDATA_TX_INTERVAL{std::chrono::seconds{60}};` -> `auto GETDATA_TX_INTERVAL{60s};`

As I could not see any apparent reason to keep it in microseconds.